src/GrpcProtos/capture.proto

```diff
@@ -324,17 +344,20 @@ message ClientCaptureEvent {
     AddressInfo address_info = 16;
     ApiEvent api_event = 9;
     CallstackSample callstack_sample = 1;
+    CaptureStarted capture_started = 24;
     FunctionCall function_call = 2;
     GpuJob gpu_job = 3;
     GpuQueueSubmission gpu_queue_submission = 4;
     InternedCallstack interned_callstack = 5;
     InternedString interned_string = 18;
     InternedTracepointInfo interned_tracepoint_info = 19;
     IntrospectionScope introspection_scope = 20;
+    ModulesSnapshot modules_snapshot = 25;
```

Comment:
Isn't `module_update_event` before `modules_snapshot` because of the space? Same below.

Reply:
sort disagrees
```
$ echo "modules_snapshot
> module_update" | sort
modules_snapshot
module_update

```

Reply:
Which means that I disagree with sort for this...